src/Onboarding/Setup/templates/index.html.php

```diff
@@ -58,7 +58,7 @@
 												: $this->image( 'paypal@2x.min.png' ),
 							'icon_alt'    => esc_html__( 'PayPal', 'give' ),
 							'title'       => esc_html__( 'Connect to PayPal', 'give' ),
-							'description' => esc_html__( 'PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant for many of the world\'s top NGOs. Accept PayPal, Credit and Debit Cards, and more using PayPal’s Smart Buttons without any added platform fees.', 'give' ),
+							'description' => esc_html__( 'PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant for many of the world\'s top NGOs. Allow donors to log in to their PayPal account to complete the donation.', 'give' ),
```

Comment:
@Benunc not sure I like this edit. They don't actually have to log into their PayPal account if they don't have one or want to.

Reply:
@DevinWalker is that true in 2.8.0, though? as of 2.9.0 the removed line will be true. right now, it's still just PayPal Standard.

Reply:
It is true in 2.8.0 as well @Benunc

Reply:
@DevinWalker credit and debit cards are only possible for a small percentage of users, in my experience. Smart buttons are not at all possible. I much prefer the second edit for setting expectations that this gateway currently redirects to PayPal for the donation to be complete.

Reply:
@DevinWalker Does this work? I kept the mention of credit and debit cards, but removed the mention of smart buttons which to my knowledge won't be available until 2.9.0:
> PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant for many of the world\'s top NGOs. Accept PayPal, Credit and Debit Cards without any added platform fees.

Reply:
Resolving this as it was approved by Devin in Slack.